Description: fix test failure due to hash randomization in perl 5.18
Origin: vendor
Bug-Debian: http://bugs.debian.org/711607
Author: CSILLAG Tamas <cstamas@cstamas.hu>
Reviewed-by: gregor herrmann <gregoa@debian.org>
Last-Update: 2013-06-09

--- a/t/01-functional.t
+++ b/t/01-functional.t
@@ -90,7 +90,7 @@
 get '/myprivs' =>  sub {
     my $self = shift;
     my @privs = $self->privileges();
-    my $priv = join(':',@privs);
+    my $priv = join(':', sort @privs);
     $self->render(text=>$priv);
 };
 my $t = Test::Mojo->new;
